lib/package/base/server-request/channels/channels_api_service.dart

```diff
@@ -243,6 +246,29 @@ class ChannelsApiService {
     return false;
   }
 
+  Future deleteChannelMessage(
+      String orgId, String channelId, String messageId, String userId) async {
+    try {
+      final res = await _api.delete(
+        '/v1/$orgId/messages/$messageId/?user_id=$userId&channel_id=$channelId',
```

Comment:
I don't understand this API call. 
Can you confirm this API endpoint from the documentation because I cannot see it? Besides shouldn't the endpoint be something like `/v1/$orgId/channels/$channelId/messages/$messageId...`?

Reply:
Alright awesome